src/patternfly/components/Button/button.scss

```diff
@@ -110,6 +110,12 @@
   line-height: var(--pf-c-button--LineHeight);
   text-align: center;
   white-space: nowrap;
+  /* stylelint-disable */
```

Comment:
You can drop the prefixes here, we should have autoprefixer for this. There shouldn't be too many cases where you need to disable the linter, if you find you need to then we should talk about it to confirm it's correct.

Reply:
That makes sense, fixed.